app/Main.hs

```diff
@@ -51,19 +54,94 @@ randomNums x = do
 getAnswer :: String -> Maybe String
 getAnswer x = do
   y <- readMaybe x
-  Prelude.lookup y baseNumbers
+  countInJapanese y
 
-numberWang :: IO ()
-numberWang = do
-  num <- randomNums 1
+-- `numberWang` generates a digit number (1 to 5) and
+-- using that a random quiz number.
+-- If the guess is equal to the actual translation THAT'S NUMBERWANG!
+numberWang :: Int -> IO ()
+numberWang x = do
```

Comment:
Up to which digit do we handle in `numberWang`? We should probably warn user if they enter a larger digit count than we can handle.